spec/apruve/resources/invoice_return_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'spec_helper'
+
+describe Apruve::InvoiceReturn do
+  before :each do
+    Apruve.configure('f5fbe71d68772d1f562ed6f598b995b3', 'local')
+  end
+
+  let (:id) { '89ea2488fe0a5c7bb38aa7f9b088874a' }
+  let (:invoice_id) { '89ea2488fe0a5c7bb38aa7f9b088874b' }
+  let (:amount_cents) { 12345 }
+  let (:currency) { 'USD' }
+  let (:reason) { 'DAMAGED' }
+
+  let (:invoice_return) do
+    Apruve::InvoiceReturn.new(
+        id: id,
+        invoice_id: invoice_id,
+        amount_cents: amount_cents,
+        currency: currency,
+        reason: reason
+    )
+  end
+  subject { invoice_return }
+
+  it { should respond_to(:id) }
+  it { should respond_to(:invoice_id) }
+  it { should respond_to(:amount_cents) }
+  it { should respond_to(:currency) }
+  it { should respond_to(:uuid) }
+  it { should respond_to(:reason) }
+  it { should respond_to(:merchant_notes) }
+  it { should respond_to(:created_by_id) }
+  it { should respond_to(:created_at) }
+  it { should respond_to(:updated_at) }
+
+  describe '#find' do
+    describe 'success' do
```

Comment:
Within a unit spec for a method, RSpec allows for [contexts](http://www.betterspecs.org/#contexts) to describe a particular scenario under scrutiny in the test. So for something like this:

```ruby
describe '#find'
  context 'a successful response'
    it 'should successfully get an invoice return'
      # Invoice Return expectations here
      ...
```

We've got several specs in our code that adhere to the pattern of nested describes, just my personal preference is to use contexts.